Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -200,59 +200,31 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
-def is_tim_content(packs_in_pr: set[str], pr_files: list[str]) -> bool:
+def is_tim_content(pr_files: list[str]) -> bool:
     """
     This is where the actual search for feed:True or relevant tags or categories are being searched
     according to the login in is_tim_reviewer_needed
 
     Arguments:
-    - packs_in_pr: packs that are part of the PR
+    - pr_files: List[str] The list of files changed in the Pull Request.
 
     Returns: returns True or False if tim reviewer needed
     """
     integrations_checked = []
     for file in pr_files:
-        print(f'file is: {file}')
         integration = BaseContent.from_path(CONTENT_PATH / file)
-        print(f'Integration is: {integration}')
-        if "Integrations" in file and integration not in integrations_checked:
-            if not isinstance(integration, Integration):
-                continue
-            if integration.is_feed:
-                return True
-            integrations_checked.append(file)
-            print(f'Integration checked are: {integrations_checked}')
-            pack = integration.in_pack
-            tags = pack.tags
-            categories = pack.categories
-            if TIM_TAGS in tags or TIM_CATEGORIES in categories:
-                return True
-    return False
+        if not isinstance(integration, Integration) or integration.path in integrations_checked:
+            continue
 
-    # for file in pr_files:
-    #     if file.endswith("yml"):
-    #         integration = BaseContent.from_path(Path(file))
-    #         try:
-    #             if isinstance(integration, Integration) and integration.is_feed:
-    #                 return True
-    #         except Exception as error:
-    #             print(f'Received error: {error}\nwhen trying to find pack {integration.path}')
-    #         pack = integration.in_pack
-    #         tags = pack.tags
-    #         categories = pack.categories
-    # for pack in packs_in_pr:
-    #     pack_object = BaseContent.from_path(CONTENT_PATH / pack)
-    #     try:
-    #          if isinstance(pack_object, Pack):
-    #              for integration in pack_object.content_items.integration:
-    #                  if integration.is_feed:
-    #                      return True
-    # pack_metadata = get_pack_metadata(pack)
-    # tags = pack_metadata.get("tags")
-    # categories = pack_metadata.get("categories")
-    # if TIM_TAGS in tags or TIM_CATEGORIES in categories:
-    #     return True
-    # return False
+        if integration.is_feed:
+            return True
+        integrations_checked.append(integration.path)
```

Comment:
This should be before the if on line 219

Reply:
It doesn't really matter - if the integraition is feed, the function will return true and exit, and if it's not true, i'll continue the iteration.
but I moved it before the IF

Reply:
>but I moved it before the IF

Are you sure? it is still there

Reply:
![image](https://github.com/demisto/content/assets/85438368/a2cfea2f-3211-4774-a112-8a14985b119b)